stable/yugaware/templates/configs.yaml

```diff
@@ -151,17 +151,6 @@ data:
       location / {
         proxy_pass http://{{ eq .Values.ip_version_support "v6_only" | ternary "[::1]" "127.0.0.1" }}:9000;
       }
-
-      location ~ "^/proxy/(.*.svc.cluster.local):([0-9]{4,6})/(.*)$" {
-        resolver {{ eq .Values.ip_version_support "v6_only" | ternary "[::1]:53" "127.0.0.1:5353 ipv6=off"  }};
```

Comment:
We need to test the new internal YW proxy logic with IPv6 (I'm not worried about the format of the proxy request since it used DNS names, but I am not sure about the resolver). @iSignal thoughts (since you know more about this than I do)?

Reply:
This change should be independent of possible IPv6 resolver issues because it is essentially removing the unused /proxy link. For IPv6, we can get feedback from a run in an IPv6 k8s cluster to isolate possible issues but that shouldn't affect this diff.